Javascript-2/Week-1/Movie App/script.js

```diff
@@ -0,0 +1,197 @@
+  commentSection.className = "comment-section";
+
+  const commentInput = document.createElement("input");
+  commentInput.type = "text";
+  commentInput.value = "";
+  commentInput.placeholder = "Add a Comment";
+  commentInput.style.color = "white";
+
+  const commentButton = document.createElement("button");
+  commentButton.className = "comment-button";
+  commentButton.innerHTML = "Submit";
+
+  const comments = document.createElement("div");
+  comments.className = "comments";
+  const commentUl = document.createElement("ul");
+
+  commentButton.addEventListener("click", () => {
+    if (commentInput.value) {
+      const comment = document.createElement("li");
+      comment.textContent = commentInput.value;
+      commentUl.appendChild(comment);
+      commentInput.value = "";
+    }
+    comments.appendChild(commentUl);
+  });
+
+  commentSection.appendChild(commentInput);
+  commentSection.appendChild(commentButton);
+  commentSection.appendChild(comments);
+
+  return commentSection;
+}
+
+function addEventListeners(readMoreBtn, movieDetails, stars) {
+  readMoreBtn.addEventListener("click", function () {
+    this.classList.toggle("active");
+    if (movieDetails.style.display === "block") {
+      movieDetails.style.display = "none";
+    } else {
+      movieDetails.style.display = "block";
+    }
+  });
+
+  const starClassActive = "rating__star fas fa-star";
+  const starClassInactive = "rating__star far fa-star";
+  stars.forEach((star, clickedStarIndex) => {
+    star.onclick = () => {
+      stars.forEach((star, starIndex) => {
+        if (starIndex <= clickedStarIndex) {
+          star.className = starClassActive;
+        } else {
+          star.className = starClassInactive;
+        }
+      });
+    };
+  });
+}
+
+//----------------------------------------------------------
+
+function createMoviesPage() {
+  nowStreaming.innerHTML = "";
+  movies.forEach((movie) => {
+    const movieElement = createMovieCard(movie);
+    nowStreaming.appendChild(movieElement);
+  });
+}
+createMoviesPage();
+
+// JS 2
+const searchBtn = document.getElementById("search-icon");
+
+// function to search
+const inputField = document.getElementById("keyword-search");
+searchBtn.addEventListener("click", () => {
+  const inputStr = inputField.value.toLowerCase();
+  nowStreaming.innerHTML = "";
+  let found = false; // Flag to check if any movie is found
+
+  movies.forEach((movie) => {
+    if (
+      movie.title.toLowerCase().includes(inputStr) ||
+      movie.description.toLowerCase().includes(inputStr)
+    ) {
+      found = true;
+      const searchMovieElement = createMovieCard(movie);
+      nowStreaming.appendChild(searchMovieElement);
+    }
+    if (!found) {
+      nowStreaming.innerHTML = `<p>Not Found</p>`;
+    }
+  });
+});
+
+const login = document.getElementById("loginButton");
+const loginPage = document.getElementById("logIn");
+
+login.addEventListener("mouseover", function onclick() {
```

Comment:
```suggestion
login.addEventListener("mouseover", function onMouseOver() {
```
It's confusing to have a function called `onclick` for a `mouseover` event.